server/mvcc/kvstore.go

```diff
@@ -52,9 +52,11 @@ const (
 
 var restoreChunkKeys = 10000 // non-const for testing
 var defaultCompactBatchLimit = 1000
+var minimumBatchInterval = 10 * time.Millisecond
```

Comment:
Should rename to `defaultBatchInterval`.  

By 'minimum,' we mean that the interval should be adjusted to 10ms if users set a smaller value, but clearly, the current implementation does not reflect this.

Reply:
Fixed in pull request #18495: <https://github.com/etcd-io/etcd/pull/18495> 😃